cache/config/config.go

```diff
@@ -0,0 +1,8 @@
+package config
+
+import "github.com/moby/buildkit/util/compression"
+
+type RefConfig struct {
```

Comment:
Is there a use case that required a separate packages for this definition?

Reply:
Import cycle from cache->solver->cache

Reply:
Ok, let's figure this out later. It's more about what the dependency between solver and cache pkg should be.